src/mctpd.c

```diff
@@ -97,14 +93,15 @@ struct role {
 static const struct role roles[] = {
 	[ENDPOINT_ROLE_UNKNOWN] = {
 		.role = ENDPOINT_ROLE_UNKNOWN,
+		.conf_val = "Unknown",
 	},
 	[ENDPOINT_ROLE_BUS_OWNER] = {
 		.role = ENDPOINT_ROLE_BUS_OWNER,
-		.conf_val = "bus-owner",
+		.conf_val = "BusOwner",
```

Comment:
I was more intending that you add a separate `const char *dbus_val` to `struct role`. You would then use this for the dbus-value parsing.

Reply:
Because the possible value of `Mode` in configuration `/conf/mctpd.conf` is change to `BusOwner`, `Endpoint` and `Unknown` as the change in mctpd.conf.
Keeping `conf_val` and its` values is unnecessary as my opinion.

Reply:
> The configureable values of `conf_val` have to be changed also.

Yep, that's the issue. This change will expose dbus-style names which are not convention for a toml-style configuration. We'll need to keep those separate.